### ISSUE TITLE
Could this Project Do all Repos and not individual

### ISSUE THREAD

**Issue by MarkTimson-BIDev** [jeffpriz/devops-pr-stats#29]
Would like to see the scope of functionality extend over all repos in the devops service, and add filters to the report to filter specific repos, instead of report in single repo. Is this possible at all?

**Comment by jeffpriz** [comment]
Yeah, this is still on my wish list too, it's been asked for many times.  It's a bit of a re-work on things and I haven't had a ton of time of enhance this. but yes... Its still one of the main things I want to accomplish
